Resources/Prototypes/Entities/Clothing/Head/helmets.yml

```
@@ -188,3 +193,71 @@
   - type: IngestionBlocker
   - type: TemperatureProtection
     coefficient: 0.005    
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHelmetRedKnight
+  name: red knight helmet
+  description: A royal knight's favorite hat. If you hold it upside down it's actually a bucket.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/redknighthelm.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/redknighthelm.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.8
+  - type: IngestionBlocker
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHelmetBlueKnight
+  name: blue knight helmet
+  description: A royal knight's favorite hat. If you hold it upside down it's actually a bucket.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/blueknighthelm.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/blueknighthelm.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.8
+  - type: IngestionBlocker
```

Since these mostly share the same values you can parent the first helmet and remove the armor/ingestion blocker from the rest.

```suggestion
- type: entity
  parent: ClothingHeadHelmetRedKnight
  id: ClothingHeadHelmetBlueKnight
  name: blue knight helmet
  description: A royal knight's favorite hat. If you hold it upside down it's actually a bucket.
  components:
  - type: Sprite
    sprite: Clothing/Head/Helmets/blueknighthelm.rsi
  - type: Clothing
    sprite: Clothing/Head/Helmets/blueknighthelm.rsi
```